core/src/main/scala/org/apache/spark/storage/RDDInfo.scala

```diff
@@ -28,20 +28,10 @@ class RDDInfo(
     val numPartitions: Int,
     var storageLevel: StorageLevel,
     val parentIds: Seq[Int],
-    val callSite: CallSite,
+    val callSite: CallSite = CallSite.empty,
```

Comment:
This is a reasonable way to address the compilation failure, but I think this is also changing the API in a binary-incompatible way? MiMa will tell us. If so then actually we will need the new constructor to not take a final optional arg. Or, adjust callers to not actually need the optional arg and use the existing constructor, passing empty if needed.